utils/class_distribution.py

```diff
@@ -0,0 +1,98 @@
+import os
+import cv2
+import numpy as np
+from rich.console import Console
+from rich.table import Table
+from collections import defaultdict
+from typing import Dict, Optional
+import typer
+
+
+# We need to add the unetpp package to the system path so that we can import it
+import sys
+from pathlib import PurePath
+
+unetpp_path = PurePath(__file__).parent.parent
+sys.path.append(str(unetpp_path))
+
+from utils.preparation_data import CityScapesClasses
+
+
+app = typer.Typer()
+
+
+def get_class_distribution(
+    input_path: str, classes: Dict[int, str], suffix: str = ""
```

Comment:
Can we add a default suffix? The one we use from the Cityscapes dataset perhaps?